docs/contributor/MANAGING_ISSUES.md

```diff
@@ -0,0 +1,56 @@
+---
+id: managing-issues
+sidebar_label: Managing Issues
+title: Managing Issues
+---
+
+Please note that this document serves as a guide for how you might manage issues. As you become more familiar with the codebase and how it's supposed to behave you'll be able to skip steps or do things out of order as you see fit. For example you may be able to identify if a bug report is "working as intended" or you might recognize an issue as a duplicate without having a completely filled out issue. In such cases you can forgo the back-and-forth and just skip to the closing steps.
+
+Use your best judgement when triaging issues, but most of all remember to be kind, friendly, and encouraging when responding to users.
+
+## Bug Reports
+
+### Ensuring the Issue is Complete
+
+#### "Simple" bug
+
+A simple bug is a bug that can be reproduced in a single TS file plus an ESLint config (and possibly a tsconfig) - i.e. an issue reproducible in our web playground. The vast, vast majority of bug reports will fall into this category.
+
+Before you being reviewing a simple bug report you should first ensure that there is a complete, isolated reproduction case provided by the reporter. This means that there should be a playground link which shows the issue being reported. Whilst a playground link is definitely preferred; it is not explicitly required if there is code and config in the issue that can be pasted into the playground to reproduce the issue. If you cannot reproduce the issue in the playground, remove the "triage" tag, add the "awaiting response" tag, and ask the user to create an isolated reproduction in the playground and comment back when they have one. For example:
+
+> Thanks for the report `@reporter`. Unfortunately I couldn't reproduce the issue using the code you provided. Could you please create an isolated reproduction in our playground (https://typescript-eslint.io/play/) and comment back when you've got one? We prefer an isolated reproduction so that we as volunteer maintainers can quickly reproduce the issue and more easily find the cause.
```

Comment:
Aside: one of the projects on my shortlist is to have a `.github/messages.json` file or similar that can power a browser extension to fill these in quickly. Maybe such a thing exists already, or maybe I'll have to write it.

Reply:
GitHub saved replies? (I have a deja vu that we talked about it...)

Reply:
Yes! Thank you haha, that's the thing. 

And they have good docs now! https://docs.github.com/en/get-started/writing-on-github/working-with-saved-replies/using-saved-replies

Reply:
I have a few saved replies myself yeah. I wonder if we can just add them to a wiki to allow us to quickly share them between one another?

Reply:
https://github.com/JoshuaKGoldberg/refined-saved-replies

I feel like a hacker again 😎 🥲

Reply:
https://github.com/typescript-eslint/typescript-eslint/issues/5547